docs/core/linux-prerequisites.md

```diff
@@ -20,25 +20,29 @@ This article shows the dependencies needed to develop .NET Core applications on
 * [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
 * [Visual Studio Code](https://code.visualstudio.com/)
 
+> [!NOTE]
+> The .NET Core SDK (.dotnet.sdk.x.x.x) package is not required for production servers/environments. Only the .NET Core runtime package (.dotnet.runtime.x.x.x) is needed for apps deployed to production environments. The .NET Core runtime is deployed with Standalone apps as part of the deployment, however, it must be deployed for Shared apps as a seperate deployment. 
```

Comment:
What are Standalone and Shared apps? Are you talking about [framework-dependent and stand-alone deployments](https://docs.microsoft.com/en-us/dotnet/core/deploying/)? I think different articles should use the same terminology.

Reply:
Yes, that change had been made earlier. Not sure what happened!